OpenRA.Mods.Common/Projectiles/Railgun.cs

```diff
@@ -153,14 +153,17 @@ void CalculateVectors()
 
 			// An easy vector to find which is perpendicular vector to forwardStep, with 0 Z component
 			LeftVector = new WVec(ForwardStep.Y, -ForwardStep.X, 0);
-			LeftVector = 1024 * LeftVector / LeftVector.Length;
+			if (LeftVector.Length != 0)
```

Comment:
Using `LengthSquared` for these will save an unnecessary square root

Reply:
Done.